eZ/Publish/Core/MVC/Symfony/Cache/Http/LocationAwareStore.php

```diff
@@ -1,255 +0,0 @@
-<?php
```

Comment:
Well, the warning should say which class replaces it. The class works, it's just not used anymore. If somebody has a service that uses it, it will throw the warning during container compilation.

Unless you're really against it, I prefer it that way. Thank you ;)